### PR TITLE
[risk=no] Springupgrade 06 04 2024

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**

### CODE REVIEW

File: public-api/src/main/java/org/pmiops/workbench/interceptors/CorsInterceptor.java [hunk 1]
```diff
@@ -1,18 +1,18 @@
 package org.pmiops.workbench.interceptors;
```
Comment (yonghaoy) [inline public-api/src/main/java/org/pmiops/workbench/interceptors/CorsInterceptor.java]: this file is nolonger needed. We might need to delete this

File: .circleci/config.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ version: 2.1
 
 defaults: &defaults
   docker:
-    - image: allofustest/workbench:aou-db-image-8
+    - image: us.gcr.io/broad-dsp-gcr-public/workbench:buildimage-0.0.28
```
Comment (yonghaoy) [inline .circleci/config.yml:5]: will you use this image?

File: public-api/src/main/java/org/pmiops/workbench/RootController.java [hunk 3]
```diff
@@ -8,6 +8,7 @@ public class RootController {
 
   @RequestMapping("/")
   public String index() {
+    System.out.println("~~~~~~~!!!!!RootController");
```
Comment (yonghaoy) [inline public-api/src/main/java/org/pmiops/workbench/RootController.java]: remove

**Review by yonghaoy — APPROVED** [review]
Congrats! 
LG as long as it works in local.

File: .circleci/config.yml (hunk 2 above)
Comment (SrushtiGangireddy) [reply .circleci/config.yml:5]: I want to use this image till we get it to test and then will try to generate our own and see if it works.

**Pull request merged**